validator/accounts/accounts_enable_disable.go

```diff
@@ -182,7 +182,7 @@ func EnableAccountsCli(cliCtx *cli.Context) error {
 		}); err != nil {
 			return err
 		}
-		log.WithField("publicKeys", allAccountStr).Info("Accounts enabled")
+		log.WithField("publicKeys", allAccountStr).Info("Accounts are enabled. PLEASE RESTART YOUR VALIDATOR NODE for those updates to take effect")
```

Comment:
```suggestion
		log.WithField("publicKeys", allAccountStr).Info("Accounts are enabled. Please restart your validator node for those updates to take effect")
```

Please no caps like that